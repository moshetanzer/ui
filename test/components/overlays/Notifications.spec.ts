import { describe, it, expect } from 'vitest'
import { UNotifications } from '#components'
import type { TypeOf } from 'zod'
import ComponentRender from '../component-render'

describe('Notifications', () => {
  it.each([
    ['basic case', {}]
    // // @ts-ignore
  ])('renders %s correctly', async (nameOrHtml: string, options: TypeOf<typeof UNotifications.props>) => {
    const html = await ComponentRender(nameOrHtml, options, UNotifications)
    expect(html).toMatchSnapshot()
  })
})
