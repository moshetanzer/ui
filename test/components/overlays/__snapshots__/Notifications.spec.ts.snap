// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Notifications > renders basic case correctly 1`] = `
"<!--teleport start-->
<!--teleport end-->"
`;
